src/components/connection-modal/connection-modal.css

```diff
@@ -225,14 +225,14 @@
     display: flex;
     justify-content: center;
     align-items: center;
+    padding: 1rem .5rem;
```

Comment:
Adds padding to right and left, puts scratch-link-help padding here so it's more generally usable